hetio/neo4j.py

```diff
@@ -8,14 +8,28 @@
 import py2neo.packages.httpstream
 import pandas
 
+from tqdm import tqdm
+from functools import reduce
```

Comment:
The functools and operator imports should go in the first group of  standard library imports. `tqdm` import should go with second group (third party imports). See https://www.python.org/dev/peps/pep-0008/#imports.